(Feature) Feature/amount decimals input

kbardi opened the pull request:
This PR closes [#166872971](https://www.pivotaltracker.com/story/show/166872971), by:

* Updating Amount page, removing mask from input, and fixing caret position with custom keyboard and computer keyboard. 
* Updating some styles.
* Showing placeholder by default.
* Adding integer amount instead of decimal value by default

**Extra:**
![sf2evgEgdr](https://user-images.githubusercontent.com/4523375/60245204-79587c00-9892-11e9-8e94-bbcc2f80f4c2.gif)

fernandomg commented:
> I'm using the NumPad component, but the numbers are added not in the order expected. And also the `0.00` is still there.
> 
That was fixed. :rocket: 

@kbardi, a new issue I found, is that I can enter commas in any place:
![image](https://user-images.githubusercontent.com/3315606/60360182-81f3a400-99b1-11e9-9a62-238cfcdbfab3.png)

kbardi commented:
> @kbardi The confirm screen shows wrong amount.
> It seems like a wei/gd issue
> Please also add a unit test for this since it's critical.

I've added some test and I fixed the error with confirmation screen amount

sirpy commented on src/lib/wallet/utils.js:
```diff
@@ -25,9 +25,9 @@ export const weiToGd = (wei: number): number => (wei * Math.pow(0.1, DECIMALS)).
+ * @returns {string}
  */
-export const gdToWei = (gd: number): number => (gd * Math.pow(10, DECIMALS)).toFixed(0)
+export const gdToWei = (gd: string): string => (gd * Math.pow(10, DECIMALS)).toFixed(0)
```
why string?

kbardi replied:
Because the input is a text type input, so the parameter we have is a string...anyway it works fine...and it returns a string because we use `toFixed()`, that convert the number to string

kbardi commented:
> If delete with numPad backspace, with the cursor at the leftmost position value starts to duplicate:
> 
> ![deleting-leftmost-number](https://user-images.githubusercontent.com/3315606/60472569-baa8ad00-9c3f-11e9-988c-dc65f909f802.gif)
> 
> If I start with zeros, I can't add decimals, but I can add up to 2 other whole numbers:
> 
> ![leading-zeroes-3-digits-max](https://user-images.githubusercontent.com/3315606/60472695-4b7f8880-9c40-11e9-8a5d-ead0e40b2193.gif)

Fixed with last commit @fernandomg

sirpy commented:
@kbardi solve and merge
